src/User.java

```diff
@@ -0,0 +1,11 @@
+import java.util.ArrayList;
+
+public class User {
+    ArrayList<String> name = new ArrayList<>();
```

Comment:
Change type to List.
We should use interfaces as much as we can ( instead of implementations)